Source/RequestBuilder.swift

```diff
@@ -342,22 +342,25 @@ import AdSupport
         return nil
     }
 
-    func openrtbRegs() -> [AnyHashable: Any]? {
+    func openrtbRegs() -> [AnyHashable: Any] {
 
         var regsDict: [AnyHashable: Any] = [:]
+        var ext: [AnyHashable: Any] = [:]
 
-        let gdpr = Targeting.shared.subjectToGDPR
-
-        if gdpr == true {
-            regsDict["ext"] = ["gdpr": NSNumber(value: gdpr).intValue] as NSDictionary
+        if Targeting.shared.subjectToGDPR == true {
+            ext["gdpr"] = 1
         }
+        
+        ext["us_privacy"] = StorageUtils.iabCcpa()
```

Comment:
Is this object `ext["us_privacy"]` created even if the ccpa is empty or nil?

Reply:
If `StorageUtils.iabCcpa()` returns `nil` or empty string the `ext["us_privacy"]` will not be created

Check tests: `testPostDataWithEmptyCCPA` and `testPostDataWithoutCCPA`